src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -57,11 +71,30 @@ public Climber() {
     m_rightPIDController.setFF(0);
 
     Utilities.burnMotor(m_rightMotor);
+    m_leftLimitSwitch = m_leftMotor.getForwardLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen);
+    m_rightLimitSwith = m_rightMotor.getForwardLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen);
+    m_state = State.UNINITIALIZED;
+    initialize();
+  }
+
+  private void initialize() {
+    m_leftPIDController.setReference(ClimberConstants.kInitializingSpeed, ControlType.kVelocity);
+    m_rightPIDController.setReference(ClimberConstants.kInitializingSpeed, ControlType.kVelocity);
```

Comment:
It looks likely to me that this will cause the climbers to extend rather than retract. Should `kInitializingSpeed` be negative?